guides/docs/channels.md

```diff
@@ -47,14 +77,21 @@ The message flow looks something like this:
 
 ### Socket Handlers
 
-Phoenix holds a single connection to each client and multiplexes its channel sockets over that one connection. Socket handlers, such as `lib/hello_web/channels/user_socket.ex`, are modules that authenticate and identify a socket connection and allow you to set default socket assigns for use in all channels.
+Socket handlers, such as `lib/hello_web/channels/user_socket.ex` in the example app below, are called when Phoenix is setting up a channel connection.
+Connections to a given URL will all use the same socket handler, based on your endpoint configuration.
+But that handler can be used for setting up connections on any number of topics.
+
+Within the handler, you can authenticate and identify a socket connection and set default socket assigns.
 
 ### Channel Routes
 
-These are defined in Socket handlers, such as `lib/hello_web/channels/user_socket.ex`, which makes them distinct from other routes. They match on the topic string and dispatch matching requests to the given Channel module. The star character `*` acts as a wildcard matcher, so in the following example route, requests for `sample_topic:pizza` and `sample_topic:oranges` would both be dispatched to the `SampleTopicChannel`.
+Channel routes are defined in socket handlers, such as `lib/hello_web/channels/user_socket.ex` in the example app below.
+They match on the topic string and dispatch matching requests to the given Channel module.
+
+The star character `*` acts as a wildcard matcher, so in the following example route, requests for `elixir:ecto` and `elixir:phoenix` would both be dispatched to the `ElixirTopicChannel`.
```

Comment:
It may be worth changing the example here, as readers may be confused by these topics due to their usage elsewhere in the framework.

Perhaps we could go with `room:lobby`, `room:123` and `RoomTopicChannel instead?

Reply:
Yep - fixed in latest commit.